### DISCUSSION TITLE
How to change the default DNS server

### DISCUSSION THREAD

**Discussion by dark-cli** [wg-easy/wg-easy#1074] [Q&A] [answered]
I can find, a way to change the DNS, I do have my own dns-server running on the same server where I am hosting the wg-wasy, It is my first time with wg. I don't know if the settings can or can not be change manual from the server.

**Comment by eandersons** [comment, marked as answer]
Have you read the section [Options](/wg-easy/wg-easy#options)?
There the environment variable `WG_DEFAULT_DNS` can be found:

> DNS server clients will use. If set to blank value, clients will not use any DNS.

**Reply by dark-cli** [reply]
Yup, thanks. My bad
